exonum/src/blockchain/transaction.rs

```diff
@@ -40,7 +39,73 @@ pub type ExecutionResult = Result<(), ExecutionError>;
 #[derive(Clone, Debug, PartialEq)]
 pub struct TransactionResult(pub Result<(), TransactionError>);
 
-/// Transaction processing functionality for `Message`, which allows applying authenticated, atomic,
+/// Data transfer object for transaction.
+/// This structure is used to send api info about transaction,
+/// and take some new transaction into pool from user input.
+#[derive(Serialize, Deserialize)]
+pub struct TransactionMessage {
```

Comment:
`Message<Box<dyn Transaction>>` would work (because i left bound only on impl body), 
but i prefer to keep `Message<T>` single responsible (keep raw message near payload for internall use).

I do not like to assign the responsibility of the "creating user representation of transaction" to `Message` part.